### ISSUE TITLE
Deduplicate components found in sca and container scanning

### ISSUE THREAD

**Issue by timbastin** [l3montree-dev/devguard#175]
<img width="1382" alt="Bildschirmfoto 2024-09-24 um 09 41 08" src="https://github.com/user-attachments/assets/2f52c18c-e30a-4dfb-a47d-fea6abc6b0bc">

**Comment by Hubtrick-Git** [comment]
Isn't that the same Issue as Issue #197 Deduplication of CVEs ?

**Comment by timbastin** [comment]
@Hubtrick-Git  yes you are right! Thanks a lot for pointing out. Closing this in favor of #197
